hydromt_sfincs/workflows/merge.py

```diff
@@ -88,9 +88,20 @@ def merge_multi_dataarrays(
         method = "bilinear"
 
     if da_like is not None:  # reproject first raster to destination grid
-        da1 = da1.raster.reproject_like(da_like, method=method).load()
+        da1 = da1.raster.clip_bbox(
+            da_like.raster.box.to_crs(da1.raster.crs.to_epsg()).total_bounds, buffer=10
+        )
```

Comment:
Is this clipping required before reprojecting? If so, it would be good to add this in the reproject_like method in the core. In my previous tests it didn't seem to matter.